src/NServiceBus.Core/Pipeline/BehaviorContextStacker.cs

```diff
@@ -8,7 +8,7 @@
     /// <summary>
     /// A stack of <see cref="BehaviorContext"/>s.
     /// </summary>
-    public class BehaviorContextStacker : IDisposable
+    internal class BehaviorContextStacker : IDisposable
```

Comment:
My old style cop obsession still gets me from time to time :(

Am Donnerstag, 23. Juli 2015 schrieb Szymon Pobiega :

> In src/NServiceBus.Core/Pipeline/BehaviorContextStacker.cs
> <https://github.com/Particular/NServiceBus/pull/2784#discussion_r35366440>
> :
>
> > @@ -8,7 +8,7 @@
> >      /// <summary>
> >      /// A stack of <see cref="BehaviorContext"/>s.
> >      /// </summary>
> > -    public class BehaviorContextStacker : IDisposable
> > +    internal class BehaviorContextStacker : IDisposable
>
> Nitpicking: internal is the default
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/2784/files#r35366440>.
>